reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
<p>Edited text below:</p><p> </p><p>In order to do Batch RL training, we need to first <strong>prepare the dataset that </strong>is generated by a deployed policy. In real world scenarios, customers can collect these offline data by interacting the live environment <strong>using the already deployed </strong>agent. In this notebook, we used OpenAI gym&nbsp;<code>Cartpole-v0</code>&nbsp;as the environment to mimic a live environment and used a random policy with uniform action distribution to mimic a deployed agent. By interacting with multiple environments simultaneously, we can gather more trajectories from the environments.&nbsp;</p><p> </p><p>Here is a short introduction of the cart-pole balancing problem, where a pole is attached by an un-actuated joint to a cart, moving along a frictionless track.&nbsp;</p><ol><li><em>Objective</em>: Prevent the pole from falling over&nbsp;</li><li><em>Environment</em>: The environment used in this example is part of OpenAI Gym, corresponding to the version of the cart-pole problem described by Barto, Sutton, and Anderson [1]&nbsp;</li><li><em>State</em>: Cart position, cart velocity, pole angle, pole velocity at tip&nbsp;</li><li><em>Action</em>: Push cart to the left, push cart to the right&nbsp;</li><li><em>Reward</em>: Reward is 1 for every step taken, including the termination step</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>